packages/TorneloScoresheet/src/pages/ResultDisplay/style.ts

```diff
@@ -1,6 +1,12 @@
-import { StyleSheet } from 'react-native';
+import { Dimensions, StyleSheet } from 'react-native';
+import { TOOLBAR_HEIGHT } from '../../components/Toolbar/style';
+
+const { height } = Dimensions.get('screen');
 
 export const styles = StyleSheet.create({
+  container: {
+    height: height - TOOLBAR_HEIGHT,
```

Comment:
Could we not define this  container style globally? and just resue it everywhere

`imoprt globalStyles`

`globalStyles.container`
since we're gonna resuse it everywhere
imagine if we add a new component on top of everything wed have to change this evrywehre